scipy/stats/_stats_py.py

```diff
@@ -3334,16 +3333,13 @@ def iqr(x, axis=None, rng=(25, 75), scale=1.0, nan_policy='propagate',
         ``(25, 75)``. The order of the elements is not important.
     scale : scalar or str, optional
```

Comment:
Unrelated nit but according to the text description perhaps we should have
```suggestion
    scale : scalar or array_like or str, optional
```

Reply:
Since practically everything is array_like according to NumPy's definition, how about being a little more specific:
```suggestion
    scale : scalar or str or array_like of reals, optional
```